src/main/java/com/github/alex1304/jdash/client/AuthenticatedGDClient.java

```diff
@@ -140,6 +169,56 @@ void putExtraParams(Map<String, String> params) {
 		return fetch(new GDLeaderboardRequest(this, strategy, count));
 	}
 
+	/**
+	 * Gets the blocked users of the account that this client is logged on.
+	 *
+	 * @return a Mono emitting a list containing all blocked users.
+	 * @throws UnsupportedOperationException if this client is not logged it to any
+	 *                                       account
+	 */
+	public Mono<List<GDUserSearchData>> getBlockedUsers(){
+		return fetch(new GDBlockedUsersRequest(this));
+	}
+
+	/**
+	 * Block an user
+	 *
+	 * @param targetAccountID the ID of the target to block
+	 * @return a Mono completing empty if succeeded, an error otherwise.
+	 * @throws UnsupportedOperationException if this client is not logged it to any
+	 *                                       account
+	 * @throws IllegalArgumentException      if user isn't a registered user
+	 *                                       {@code (user.getAccountId() == 0)}
+	 */
+	public Mono<Void> blockUser(long targetAccountID){
+		return fetch(new GDUserBlockRequest(this, targetAccountID));
+	}
+
+	/**
+	 * Unblock an user
+	 *
+	 * @param targetAccountID the ID of the target to unblock
+	 * @return a Mono completing empty if succeeded, an error otherwise.
+	 * @throws UnsupportedOperationException if this client is not logged it to any
+	 *                                       account
+	 * @throws IllegalArgumentException      if user isn't a registered user
+	 *                                       {@code (user.getAccountId() == 0)}
+	 */
+	public Mono<Void> unblockUser(long targetAccountID){
+		return fetch(new GDUserUnblockRequest(this, targetAccountID));
+	}
+
+	/**
+	 * Gets the friends of the account that this client is logged on.
+	 *
+	 * @return a Mono emitting a list containing all friends.
+	 * @throws UnsupportedOperationException if this client is not logged it to any
+	 *                                       account
+	 */
+	public Mono<List<GDUserSearchData>> getPrivateFriends(){
```

Comment:
* "private friends" is a weird name, why not just `getFriends()`?
* If the server returns info on the pagination of the friends list, please return a `Mono<GDPaginator>` instead of a `Mono<List>`.

Reply:
Renamed to getFriends 14270ba

The reason why method name was `getPrivateFriends` was named after `getPrivateMessages`, but it was a bit strange to think about it again.